### PR TITLE
Shop PWA Fundamentals

### PR DESCRIPTION
**Pull request by njfamirm**
## Description

soffit PWA fundamental...

## PR Checklist

Please check if your PR fulfills the following requirements:

- [x] Open an issue for bug report or feature request and mention it to this PR.
- [x] The commit message follows our guidelines: <https://github.com/AliMD/alwatr/blob/main/CONTRIBUTING.md#commit-message-format>
- [x] All methods have proper description in `jsdocs` format with `@example` section.
- [x] Docs have been added / updated (for bug fixes / features).
- [x] I have performed a self-review of my own code.

### CODE REVIEW

File: src/elements/page-bookmarks.ts [hunk 1]
```diff
@@ -110,22 +109,7 @@ export class PageBookmarks extends AlwatrElement {
     return html`
       <ion-header translucent dir="rtl">
         <ion-toolbar>
-          <ion-segment value="bookmarks" mode="md">
```
Comment (AliMD) [inline src/elements/page-bookmarks.ts:113]: اینارو تغییر نده بزار صفحه bookmark بمونه

File: src/config.ts [hunk 2]
```diff
@@ -4,30 +4,20 @@ export const mainTabBar = [
...
-  // {
-  //   id: 'about-him',
-  //   title: 'احمد الحسن',
-  //   icon: 'person',
-  // },
   {
-    id: 'beliefs',
-    title: 'اعتقادات',
-    icon: 'chatbubbles',
+    id: 'products',
```
Comment (AliMD) [inline src/config.ts:8]: همه products ها بشه product-list

**Review by AliMD — CHANGES_REQUESTED** [review]

**Comment by njfamirm** [comment]
دیگه کجا رو تغییر بدم؟

**Comment by MM25Zamanian** [comment]
سافیت توی توضیحات اشتباهه

**Comment by njfamirm** [comment]
> سافیت توی توضیحات اشتباهه

این انتقام از الشفعه ؟ 🤣

**Pull request merged**